src/vqv.js

```diff
@@ -16,6 +16,15 @@
       #VQV!'
 */
 
-const vqv = (nome, idade) => {};
+const vqv = (nome, idade) => {
+  if (nome && idade) {
+    return `Oi, meu nome é ${nome}!
+Tenho ${idade} anos,
+trabalho na Trybe e mando muito em programação!
+#VQV!`;
+  }
```

Comment:
Sergio, caso você não queira quebrar a linha nesse formato exigido pelo template literals, você pode usar assim:

```suggestion
    return `Oi, meu nome é ${nome}!\nTenho ${idade} anos,\ntrabalho na Trybe e mando\nmuito em programação!\n#VQV!`;
  }
```

Reply:
Jack, eu até tentei fazer assim, mas da conflito no lint, dizendo que a linha tem um tamanho maior que o esperado